tests/conftest.py

```diff
@@ -2,9 +2,33 @@
 from pathlib import Path
 
 import pytest
+from qibolab._core.platform.load import PLATFORMS
+
+from qibocal.calibration.platform import create_calibration_platform
+
+TESTING_PLATFORM_NAMES = [
+    "mock",
+]
 
 
 @pytest.fixture(autouse=True)
 def cd(tmp_path_factory: pytest.TempdirFactory):
     path: Path = tmp_path_factory.mktemp("run")
     os.chdir(path)
+
+
+def set_platform_profile():
+    os.environ[PLATFORMS] = str(Path(__file__).parent / "platforms")
+
+
+def unset_platform_profile():
+    if PLATFORMS in os.environ:
+        del os.environ[PLATFORMS]
+
+
+@pytest.fixture(scope="module", params=TESTING_PLATFORM_NAMES)
+def platform(request):
+    """Dummy platform to be used when there is no access to QPU."""
+    set_platform_profile()
+    yield create_calibration_platform(request.param)
+    unset_platform_profile()
```

Comment:
To fully revert the status, you should something like the following:
```py
@pytest.fixture(scope="module", params=TESTING_PLATFORM_NAMES)
def platform(request):
    """Dummy platform to be used when there is no access to QPU."""
    old_platforms = os.environ.get(PLATFORMS)
    os.environ[PLATFORMS] = str(Path(__file__).parent / "platforms")
    yield create_calibration_platform(request.param)
    if old_platforms is not None:
        os.environ[PLATFORMS] = old_platforms
    else:
        del os.environ[PLATFORMS]
```
but there is actually an easier way
```suggestion
@pytest.fixture(scope="module", params=TESTING_PLATFORM_NAMES)
def platform(request, monkeypatch):
    """Dummy platform to be used when there is no access to QPU."""
    monkeypatch.setenv(PLATFORMS, str(Path(__file__).parent / "platforms"))
    return create_calibration_platform(request.param)
```
which is essentially doing the same, but already implemented by Pytest.

Reply:
For sure I like your proposal better than mine :)